.eslintrc.json

```diff
@@ -47,7 +47,7 @@
     "import/no-extraneous-dependencies": [
       "error",
       {
-        "devDependencies": false,
+        "devDependencies": true,
```

Comment:
I changed to `true` because I think it should be possible to import from `devDependencies` (in a test) https://github.com/import-js/eslint-plugin-import/blob/v2.24.2/docs/rules/no-extraneous-dependencies.md